crypto/cms/cms_sd.c

```diff
@@ -227,19 +228,49 @@ int ossl_cms_SignerIdentifier_cert_cmp(CMS_SignerIdentifier *sid, X509 *cert)
         return -1;
 }
 
+/* Method to map any, incl. provider-implemented PKEY types to OIDs */
+/* ECDSA and DSA and all provider-delivered signatures implementation is the same */
+static int cms_generic_sign(CMS_SignerInfo *si, int verify)
+{
+    assert(verify == 0 || verify == 1);
```

Comment:
Please replace with

```
if (!ossl_assert(verify == 0 || verify == 1))
   return -1;
```

Reply:
Fixed in https://github.com/openssl/openssl/pull/17733/commits/6a6172046810203bff7daecc2d6d0c1df2b8ce93